app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -763,6 +764,19 @@ public void onTabChanged(String tabId) {
         }
     }
 
+    /**
+     * Handles resolution callbacks.
+     */
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode,
```

Comment:
Why onActivityResult is necessary if we are not using startActivityForResult() method?